docs/01-getting-started.md

```diff
@@ -17,20 +17,26 @@ The Sleeper CLI has the following dependencies:
 ### Installation
 
 The Sleeper CLI contains Docker images with the necessary dependencies and scripts to work with Sleeper. Run the
-following commands to install the latest nightly build of the CLI.
+following commands to install the latest development version of the CLI.
 
 ```bash
-curl "https://raw.githubusercontent.com/gchq/sleeper/main/scripts/cli/install.sh" -o ./sleeper-install.sh
+curl "https://raw.githubusercontent.com/gchq/sleeper/develop/scripts/cli/install.sh" -o ./sleeper-install.sh
 chmod +x ./sleeper-install.sh
-./sleeper-install.sh
+./sleeper-install.sh develop
 ```
 
-You can also specify a version to install. This can be `main` for the latest nightly build, or a release in the
-format `v0.20.0`. These correspond to a branch or tag in the GitHub repository. You can find a list of released
-versions [here](https://github.com/gchq/sleeper/tags), and the change log [here](../CHANGELOG.md).
+This will install the latest development version. This is an untested version and may not work. When ready this is
```

Comment:
Replace "This is an untested version and may not work." with "The latest development version will not have been as fully tested as released versions and may not work as expected. Most users should use the latest released version - see [here](../CHANGELOG.md."